configs/neptune.yaml

```diff
@@ -1,135 +0,0 @@
-project: ORGANIZATION/home-credit
```

Comment:
@pranayaryal We do need those configs to parse parameters.

Reply:
What's the best way to gitignore personal configs?

Reply:
Well we usually just create neptune_local.yaml and then add that to .gitignore